docs/logs/getting-started/README.md

```diff
@@ -61,6 +61,15 @@ instance, which is then used to do the logging. The log is sent to the
 `OpenTelemetryLoggerProvider`, which is configured to export logs to
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
+Adding logging providers, obtaining `ILogger` instance, etc. could alternatively
+be done in ways, more idiomatic to the application type. For e.g. [Logging in
```

Comment:
This line sounds weird to me. Consider getting some help from native English speakers (I am not).